source/features/pagination-hotkey.tsx

```diff
@@ -21,19 +22,32 @@ const previousPageButtonSelectors = [
 function init(): void {
 	const createNextPageButton = select(nextPageButtonSelectors);
 	if (createNextPageButton) {
-		createNextPageButton.dataset.hotkey = 'ArrowRight';
+		createNextPageButton.dataset.hotkey = isRepoConversationList() ? 'ArrowRight' : 'n';
```

Comment:
Pick one

Reply:
I favor `n` / `p` that is closer to GitHub's preference.

But then, that would mean excluding the feature on `isRepoConversationList` - 3018387

Reply:
or otherwise, like you said in https://github.com/sindresorhus/refined-github/issues/4742#issuecomment-907791088

> don't feel strongly to make the change yet, unless there are already other pages using n/p on GitHub.

we could drop this pull request for now and get back to it when we are ready to make the switch.

Reply:
Ahhh that's a problem. Indeed than maybe we can leave as was, except the feature name 😰

Reply:
I do not feel strongly about simply renaming a feature. I think it can go along with the update to the feature _if it happens when it happens_:tm:.

... unless we are doing a mass-rename of many features together to follow https://github.com/sindresorhus/refined-github/pull/4745#discussion_r698044155